org-formation/100-shared-dns/README.md

```diff
@@ -34,3 +34,37 @@ This zone is intended for application-facing applications, such as Schematic.
 CDK applications will export a CloudFormation output with the AWS domain
 name for the created ALB. Create a DNS CNAME record in the appropriate
 hosted zone with a value pointing to the AWS domain name of the ALB.
+
+## Per-Account Certificates
+
+Certificates must exist in the same account they will be used in, and so we
+must create a copy of our wildcard certificate in an application account before
+the application can be deployed to that account. For example, [the list of
+accounts for `app.sagebionetworks.org`](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/100-shared-dns/_tasks.yaml#L70).
```

Comment:
i'm not a fan of specific line references in docs because it will very likely point to something else at a later time.  how about we just reference the entire file instead?

Reply:
in that case I don't think it makes sense to have a link, I'll just quote the name of the only other file in this directory